packages/lit-dev-content/site/docs/components/events.md

```diff
@@ -42,6 +42,14 @@ private _handleTouchStart(e) { console.log(e.type) }
 
 </div>
 
+You can customize event listener options by passing an object which has `handleEvent` property and a name whose options of `addEventListener` property to [event listener expressions](https://lit.dev/docs/templates/expressions/#event-listener-expressions):
```

Comment:
This sentence is confusing. Perhaps something like this:

```suggestion
If you're not using decorators, you can customize event listener options by passing an object to the event listener expression. The object must have a `handleEvent()` method and can include any the options that would normally appear in the `options` argument to `addEventListener()`.
```

Reply:
Thank you for review. I changed it.